bin/resources/GameIndex.yaml

```diff
@@ -28072,7 +28072,7 @@ SLPM-65731:
   name: "Tom Clancy's Rainbow Six 3"
   region: "NTSC-J"
 SLPM-65732:
-  name: "Akaiito"
+  name: "SuperLite 2000 Vol. 34 - Akai Ito"
```

Comment:
SLPM-65732 is the original version of ``Akai Ito``

Reply:
Reverted to base title